src/Synth/OscilGen.cpp

```diff
@@ -1864,12 +1864,12 @@ FILTER(s)
     return gain;
 }
 
-FILTER(lpsk2)
+FILTER(lpsk)
 {
     float tmp2PIf = 2.0f * PI * (1.05f-par)*64.0f;
     std::complex<float> s  (0.0f,2.0f*PI*i);
     float vOut = tmp2PIf * tmp2PIf;
-    std::complex<float> vIn = s*s + tmp2PIf*s/(4*par2+0.5f) + tmp2PIf*tmp2PIf;
+    std::complex<float> vIn = s*s + tmp2PIf*s/((par2)+(2.0f*par*par2)+0.5f) + tmp2PIf*tmp2PIf;
```

Comment:
Just a sanity check you do want par*par2 and not par2*par2, right? I would have just merged, but right now the PR doesn't compile since you changed lpsk2 to lpsk without updating some other glue code (just find the osc_lpsk2 line or check the CI failure)

Reply:
par*par2 is right because for higher freqs (par) the impact of q (par2)  should be higher(3x).
osc_lpsk2 fix is underway.